.github/workflows/unable-to-reproduce-comment.yml

```diff
@@ -0,0 +1,38 @@
+name: Add unable-to-reproduce comment
+on:
+  issues:
+    types:
+      - labeled
+
+permissions:
+  issues: write
+
+jobs:
+  add-comment-to-unable-to-reproduce-issues:
+    if: github.event.label.name == 'unable-to-reproduce'
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      GH_REPO: ${{ github.repository }}
+      NUMBER: ${{ github.event.issue.number }}
+      LABELS: more-info-needed
+      BODY: >
+        Thank you for your issue! Unfortunately, we are unable to reproduce the
+        issue you are experiencing. Please provide more information so we can
+        help you.
+
+
+        Here are some tips for writing reproduction steps: 
+          - Step by step instructions accompanied by screenshots or screencasts
+        are the best.
+          - Be as specific as possible; include as much detail as you can. 
+          - If not already provided, include:
+              - the version of GitHub Desktop you are using. 
+              - the operating system you are using
+              - any environment factors you can think of.
+              - any custom configuration you are using.
+          - If relevant and can be shared, provide the repository or code you
+        are using.
```

Comment:
This was mostly AI suggested. I just did a iteration of it. Happy to take suggestions to improve it.